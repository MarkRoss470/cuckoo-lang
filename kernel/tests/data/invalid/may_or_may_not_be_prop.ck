data Invalid.{u} : Sort u where