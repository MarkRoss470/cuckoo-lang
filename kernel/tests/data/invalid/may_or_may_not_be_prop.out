ERROR at tests/data/invalid/may_or_may_not_be_prop.ck 1:20-1:27: Inductive types must either always be in Prop or always not be in Prop, but the level
  u
could be either.